Framework/Utils/test/test_RootTreeWriter.cxx

```diff
@@ -188,14 +188,14 @@ BOOST_AUTO_TEST_CASE(test_RootTreeWriter)
   // temporary argument is still in memory
   // FIXME: check why the compiler does not detect this
   std::vector<InputRoute> schema = {
-    {InputSpec{"input1", "TST", "INT"}, 0, "input1", 0},       //
-    {InputSpec{"input2", "TST", "CONTAINER"}, 1, "input2", 0}, //
-    {InputSpec{"input3", "TST", "CONTAINER"}, 2, "input3", 1}, //
```

Comment:
@ktf Was that actually a correct usage of `timeslice` before? As far as I understand (perhaps I am wrong), time-pipeline inputs can have multiple InputRoutes, but can correspond to one and only InputSpec. So in a real DPL device, the user algorithm would see a message coming from only one of the two InputRoutes at a time. Is that correct?